### ISSUE TITLE
Aeon: html lang

### ISSUE THREAD

**Issue by rschwab** [UCSCLibrary/SiteImprove#88]
Issue: The `<html lang>` attribute is missing or is empty, or a lang attribute value is not a valid language identifier.

Page found: https://aeon.library.ucsc.edu/logon

To complete:
- [x] All pages have an html lang value
